sdk/anomalydetector/ai-anomaly-detector-rest/README.md

```diff
@@ -0,0 +1,59 @@
+# Azure AnomalyDetectorRest client library for JavaScript
+
+AnomalyDetector Rest Client
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
+- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/anomalydetector/ai-anomaly-detector-rest)
+- [Package (NPM)](https://www.npmjs.com/package/@azure-rest/ai-anomaly-detector)
+- [API reference documentation](https://docs.microsoft.com/javascript/api/@azure-rest/ai-anomaly-detector?view=azure-node-preview)
+- [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/anomalydetector/ai-anomaly-detector-rest/samples)
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 14.x.x or higher
```

Comment:
Is this from codegen? We should update it to not mention version anymore to avoid having to change it whenever we change our support matrix.   We could just use "LTS versions of Node.js"